.babelrc

```diff
@@ -1,17 +1,12 @@
 {
-  "presets": [
+  presets: [
     [
-      "@babel/env",
+      '@babel/env',
```

Comment:
Maybe it's missing a prettier conf somewhere, my editor should have changed that quote too last time

Reply:
yes, it's just prettier, the last time we changed this file it formatted it as json, now json5, not too big of a deal